wandb/sdk/internal/internal_api.py

```diff
@@ -3580,15 +3585,24 @@ def fetch_org_entity_from_entity(self, entity: str) -> Tuple[str, str]:
             },
         )
         try:
+            is_team = response["entity"]["isTeam"]
```

Comment:
Is this always there? Can you .get() it?

Reply:
actually if this crashes and goes down to the try catch i wonder what is_team is set to. maybe i should initialize is_team is false and then do .get()